src/server/socketHandlers/scConnectionHandler.js

```diff
@@ -21,16 +22,16 @@ const isTmsValid = (tmsFromDB = [], tmsFromToken = []) => {
   return true;
 };
 
-export default function scConnectionHandler(exchange) {
+export default function scConnectionHandler(exchange, sharedDataLoader) {
   return async function connectionHandler(socket) {
     // socket.on('message', message => {
     //   if (message === '#2') return;
     //   console.log('SOCKET SAYS:', message);
```

Comment:
hmm

Reply:
oh yeah, good for debuggin, but this'll be gone soon.